sonic-sensormond/tests/mock_swsscommon.py

```diff
@@ -0,0 +1,34 @@
+'''
```

Comment:
Can this be leveraged from thermalctld tests? Let's avoid as much code duplication as possible

Reply:
will check.

Reply:
there does not seem a neat way unless we specify a relative path in another directory and pull the mock implementation from there. it seems the current methodology is to write the dependent modules for each daemon. Does see like an overhead but maybe should be done across the board for all sonic-platfrom-daemons or other consumers of the code.

Reply:
Ya, without moving around the thermalctld files it would be pretty ugly, and I understand not wanting to do that as part of this feature.

Would be good to bring this up with the community and see if anyone is able/willing to address it separately.

Reply:
Agree. We can bring this up in the community meeting.